build.cmd

```diff
@@ -422,6 +417,20 @@ echo Laying out dynamically generated EventPipe Implementation
 echo Laying out ETW event logging interface
 %PYTHON% -B -Wall %__SourceDir%\scripts\genEtwProvider.py --man %__SourceDir%\vm\ClrEtwAll.man --intermediate %__IntermediatesIncDir% --exc %__SourceDir%\vm\ClrEtwAllMeta.lst || exit /b 1
 
+if /i "%__DoCrossArchBuild%"=="1" (
+    echo Laying out dynamically generated files from cross arch build
+    set __CrossCompIntermediatesIncDir=%__CrossCompIntermediatesDir%\src\inc
+    set __CrossCompIntermediatesEventingDir=%__CrossCompIntermediatesDir%\eventing
+)
```

Comment:
You can keep these as one condition (only one 'if') if all references to `__CrossCompIntermediatesIncDir` and `__CrossCompIntermediatesEventingDir` use `!` not `%`, e.g., `!__CrossCompIntermediatesEventingDir!` not `%__CrossCompIntermediatesEventingDir%`. That's the "delay evaluation" form in CMD, otherwise everything gets expanded when the "if" is processed.

Reply:
Thanks!